optuna/samplers/_nsga2.py

```diff
@@ -68,10 +68,10 @@ class NSGAIISampler(BaseSampler):
             :class:`~optuna.trial.FrozenTrial` and return the constraints. The return value must
             be a sequence of :obj:`float` s. A value strictly larger than 0 means that a
             constraints is violated. A value equal to or smaller than 0 is considered feasible.
-            If constraints_func returns more than one value for a trial, that trial is considered
-            feasible if and only if all values are equal to 0 or smaller.
+            If ``constraints_func`` returns more than one value for a trial, that trial is
+            considered feasible if and only if all values are equal to 0 or smaller.
 
-            The constraint_func will be evaluated after each successful trial.
+            The ``constraint_func`` will be evaluated after each successful trial.
```

Comment:
I found a typo for `constraints_func` although it is not included in this PR.
```suggestion
            The ``constraints_func`` will be evaluated after each successful trial.
```

Reply:
Nice catch! Thanks!